experimental/AlgebraicStatistics/src/DiscreteGraphicalModels.jl

```diff
@@ -0,0 +1,86 @@
+function bron_kerbosch(G, R, P, X)
+
+    cliques = []
+
+    if length(P) == 0 && length(X) == 0
+        
+        push!(cliques, R)
+    end
+
+    Q = P
+    Y = X
+
+    for v in Q
+
+        new_R = vcat(R, v);
+        new_Q = [p for p in Q if p in neighbors(G, v)]
+        new_Y = [p for p in Y if p in neighbors(G, v)]
+
+        cliques = vcat(cliques, bron_kerbosch(G, new_R, new_Q, new_Y))
+
+        Q = filter(i -> i != v, P)
+        Y = vcat(X, v)
+    end
+
+    return cliques
+end
+
+function maximal_cliques(G::Graph{Undirected})
+
+    max_cliques = bron_kerbosch(G, [], vertices(G), []);
+
+    unique(sort.(max_cliques))
+end
+
+
+
+
+@doc raw"""
+    graphical_model(G::Graph{Undirected}, S::GaussianRing, k_var_name::String="k")
+
+A parametric statistical model associated to an undirected graph.
+It contains an undirected graph `G`, a GaussianRing `S` where the vanishing ideal of the model naturally lives, 
```

Comment:
If you mean "GaussianRing" as in "the type name", then it should be formatted as code, i.e.:
```suggestion
It contains an undirected graph `G`, a `GaussianRing` `S` where the vanishing ideal of the model naturally lives, 
```
But perhaps you rather mean this mathematical (makes more sense to me, esp. since the type is already specified in the first line of the docstring, as part of the method signature), I suggest writing here and elsewhere:
```suggestion
It contains an undirected graph `G`, a Gaussian ring `S` where the vanishing ideal of the model naturally lives, 
```